pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/defaultvalues/ATDefaultValues.java

```diff
@@ -71,12 +68,7 @@ protected boolean setDefaultData() {
                 return false;
             }
 
-            ObjectTransformer objectTransformer = new ObjectTransformer(instance.getName());
```

Comment:
Looks like this moved to updateEsParticipantData, with minimal logging changes and no exception wrapping, since that's already done here, correct?

Reply:
Correct. This code is now in WorkflowStatusUpdate, at least temporarily. No need to add a redundant layer of exception catching for its current usage.